components/clarinet-files/Cargo.toml

```diff
@@ -13,9 +13,10 @@ bip39 = { version = "1.0.1", default-features = false }
 clarity_repl = { package = "clarity-repl", path = "../clarity-repl", default-features = false }
 libsecp256k1 = "0.7.0"
 toml = { version = "0.5.6", features = ["preserve_order"] }
-url =  { version = "2.2.2", features = ["serde"] }
+url = { version = "2.2.2", features = ["serde"] }
 tiny-hderive = "0.3.0"
 bitcoin = { version = "0.28.1", optional = true }
+clarity = { git = "https://github.com/stacks-network/stacks-blockchain.git", branch = "refactor/clarity-lib-merge" }
```

Comment:
Genuine question - should clarity_repl be the only component importing `clarity`? What do you think would be a good approach? Mostly thinking about the way we will be dealing with `WASM` moving forward.

Reply:
Do you mean `clarity_repl` imports `clarity` and then re-exports anything needed by the other components?